venues/ICLR.cc/2019/Conference/python/iclr19.py

```diff
@@ -571,10 +572,36 @@
 
 })
 
-
+# This is the Conference level Invitation for all withdrawn submissions
+withdrawn_submission_invitation = openreview.Invitation.from_json({
+    'id': WITHDRAWN_SUBMISSION_TEMPLATE_STR,
+    'readers': ['everyone'],
+    'writers': [CONFERENCE_ID],
+    'invitees': [CONFERENCE_ID],
+    'noninvitees': [],
+    'signatures': [CONFERENCE_ID],
+    'reply': {
+        'forum': None,
+        'replyto': None,
+        'readers': {
+            'description': 'The users who will be allowed to read the reply content.',
+            'values': ['everyone']
+        },
+        'signatures': {
+            'description': 'How your identity will be displayed with the above content.',
+            'values': [CONFERENCE_ID]
+        },
+        'writers': {
+            'description': 'Users that may modify this record.',
+            'values':  [CONFERENCE_ID]
+        },
+        'content': {}
```

Comment:
this isn't a problem in my opinion, but just FYI: leaving the content field empty will allow you to pass in any content fields that you want, free of restriction. it doesn't mean that you can't pass in any content (which is what you might think, intuitively)

Reply:
It is the same invitation of last year. I said that is ok because we are not going to use the UI or another signature to create notes